README.md

```diff
@@ -34,7 +34,7 @@ This document describes the vSphere Automation .NET SDK samples that use the vSp
 .NET client library. The samples have been developed to work with .NET Framework 4.5.
 
 ## Supported OnPrem vCenter Releases:
-vCenter 6.0, 6.5, 6.7, 6.7 U1 and 7.0.0.
+vCenter 6.0, 6.5, 6.7, 6.7 U1 , 7.0.0 and 7.0 U1.
```

Comment:
Please remove the space before the comma

Reply:
removed